packages/utils/telemetry-utils/src/debugLogger.ts

```diff
@@ -1,138 +0,0 @@
-/*!
- * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
- * Licensed under the MIT License.
- */
-
-import {
-	ITelemetryBaseEvent,
-	ITelemetryBaseLogger,
-	ITelemetryProperties,
-} from "@fluidframework/core-interfaces";
-import { performance } from "@fluidframework/common-utils";
-import { debug as registerDebug, IDebugger } from "debug";
-import {
-	TelemetryLogger,
-	MultiSinkLogger,
-	ITelemetryLoggerPropertyBags,
-	formatTick,
-	createChildLogger,
-} from "./logger";
-/**
- * Implementation of debug logger
- * @deprecated - DebugLogger is internal and will no longer be exported.
- */
-export class DebugLogger extends TelemetryLogger {
```

Comment:
I'm guessing you already copied this elsewhere?

Reply:
yeah. it lives with the loader which is the only thing that uses it